blocks/library/block/index.js

```diff
@@ -40,46 +41,53 @@ class ReusableBlockEdit extends Component {
 	}
 
 	startEditing() {
-		const { reusableBlock } = this.props;
+		const { reusableBlock, block } = this.props;
 
 		this.setState( {
 			isEditing: true,
 			title: reusableBlock.title,
+			attributes: block.attributes,
```

Comment:
Do we need to track all of the block's original attributes in state? Or could we simply track the state as the object which assigns into `block.attributes` as the base?

```js
this.setState( {
	attributes: {},
} );

// ... 

const attributes = { ...block.attributes, ...this.state.attributes };
```

Also has the added advantage that when we call `updateAttributes` in `save`, we're only applying the patch changes to state (likely to save a couple iterations when [aggregating into state value](https://github.com/WordPress/gutenberg/blob/030b562ed81b567806975a43900b18064ba12a8a/editor/store/reducer.js#L292) and maybe avoiding some false positives when [overriding state value for undo](https://github.com/WordPress/gutenberg/blob/030b562ed81b567806975a43900b18064ba12a8a/editor/store/reducer.js#L165)).

Reply:
👌 fixed in 5a36e93ce.